src/ape_test/provider.py

```diff
@@ -198,7 +200,10 @@ def get_virtual_machine_error(self, exception: Exception, **kwargs) -> VirtualMa
 
         elif isinstance(exception, TransactionFailed):
             err_message = str(exception).split("execution reverted: ")[-1] or None
-            err_message = None if err_message == "b''" else err_message
+
+            if err_message and err_message.startswith("b'") and err_message.endswith("'"):
+                err_message = HexBytes(literal_eval(err_message)).hex()
```

Comment:
Can you add a comment here of what the expected pattern is and why this fixed it?

Reply:
[881d446](https://github.com/ApeWorX/ape/pull/1355/commits/881d446b0c5803e7f977d4883b2dcb0866f25589)